### PR TITLE
[CommonServerPython & demistomock]  isTimeSensitive

### PR DESCRIPTION
**Pull request by shmuel44**
## Related Issues
fixes: [link](https://jira-dc.paloaltonetworks.com/browse/CIAC-10444) to the issue

## Description
- Added `is_time_sensitive` method to `CommonServerPython` 
- Added `isTimeSensitive` mock method to `demistomock`
This method will indicate whether the command reputation (auto-enrichment) is called with auto-extract=inline.

### CODE REVIEW

**Review by dorschw — APPROVED** [review]

File: Packs/Base/ReleaseNotes/1_33_53.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### CommonServerPython
+
+Added the *is_time_sensitive* method, this method will indicate whether the command reputation (auto-enrichment) is called as auto-extract=inline.
```
Comment (shmuel44) [inline Packs/Base/ReleaseNotes/1_33_53.md:6]:
```suggestion
Added the *is_time_sensitive* method, this method will indicate whether the command reputation (auto-enrichment) is called with auto-extract=inline.
```

File: Packs/Base/ReleaseNotes/1_33_53.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### CommonServerPython
+
+Added the *is_time_sensitive* method, this method will indicate whether the command reputation (auto-enrichment) is called with auto-extract=inline.
```
Comment (dorschw) [inline Packs/Base/ReleaseNotes/1_33_53.md:6]:
```suggestion
Added the *is_time_sensitive* method, indicating whether the current code is a reputation command called with auto-extract=inline.
```

File: Packs/Base/ReleaseNotes/1_33_54.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### CommonServerPython
+
+Added the *is_time_sensitive* method, this method will indicate whether the command reputation (auto-enrichment) is called with auto-extract=inline.
```
Comment (shmuel44) [inline Packs/Base/ReleaseNotes/1_33_54.md:6]:
```suggestion
Added the *is_time_sensitive* method, indicating whether the current code is a reputation command called with auto-extract=inline.
```

**Comment by ShirleyDenkberg** [comment]
@dorschw Doc review completed.

**Pull request merged**